app/views/admin/pages/index.html.erb

```diff
@@ -2,14 +2,50 @@
 
 <header class="flex items-center mb-6">
   <h2 class="crayons-title">Pages</h2>
-  <%= link_to "New page", new_admin_page_path, class: "btn btn-primary ml-auto" %>
+  <%= link_to "New page", new_admin_page_path, class: "ml-auto crayons-btn crayons-btn--s" %>
 </header>
 
-<div class="crayons-card p-6">
+<div class="crayons-card p-6 pages__table">
   <% @pages.each do |page| %>
     <div class="flex py-2 items-center">
-      <%= link_to page.title, edit_admin_page_path(page.id) %>
```

Comment:
I changed this around.

It felt counter intuitive to go to edit with the click on the link and then have a separate view button. I made the click on the link the view and then added an edit button.